app-connector-client/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server/app.js",
   "scripts": {
     "start": "node server/server",
-    "start:dev": "export DEBUG=true && node server/server test/varkes_config.json",
```

Comment:
This doesn't work on windows as we don't have the `export` keyword.

Reply:
I prefer to address that in a dedicated PR as here we want to fix the file path problems.